<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="DrawWidget" name="drawWidget" native="true"/>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="RadiusSphere">
      <property name="title">
       <string>Radius Sphere</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QSlider" name="sliderRadius">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="Elipse">
      <property name="title">
       <string>Elipse</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Rx</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Ry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Rz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="Box">
      <property name="title">
       <string>Box</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>DimX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>DimY</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>DimZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSlider_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="Dimensions">
      <property name="title">
       <string>Dimensions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderAxisX">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAxisX">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderAxisY">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAxisY">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Z</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sliderAxisZ">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelAxisZ">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QGroupBox" name="Color">
      <property name="title">
       <string>Color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,100">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>R</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderR">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>G</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderG">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>B</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSliderB">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>255</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonColor">
         <property name="text">
          <string>ColorPick</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSalvar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Matriz"/>
   <addaction name="actionInsert_Box"/>
   <addaction name="actionRemove_Box"/>
   <addaction name="actionInsert_Sphere"/>
   <addaction name="actionRemove_Sphere"/>
   <addaction name="actionInsert_Elipse"/>
   <addaction name="actionRemove_Elipse"/>
   <addaction name="actionInsert_Voxel"/>
   <addaction name="actionRemove_Voxel"/>
   <addaction name="actionXY"/>
   <addaction name="actionYZ"/>
   <addaction name="actionZX"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionInsert_Box">
   <property name="text">
    <string>Insert_Box</string>
   </property>
  </action>
  <action name="actionRemove_Box">
   <property name="text">
    <string>Remove_Box</string>
   </property>
  </action>
  <action name="actionInsert_Sphere">
   <property name="text">
    <string>Insert_Sphere</string>
   </property>
  </action>
  <action name="actionRemove_Sphere">
   <property name="text">
    <string>Remove_Sphere</string>
   </property>
  </action>
  <action name="actionInsert_Elipse">
   <property name="text">
    <string>Insert_Elipse</string>
   </property>
  </action>
  <action name="actionRemove_Elipse">
   <property name="text">
    <string>Remove_Elipse</string>
   </property>
  </action>
  <action name="actionInsert_Voxel">
   <property name="text">
    <string>Insert_Voxel</string>
   </property>
  </action>
  <action name="actionRemove_Voxel">
   <property name="text">
    <string>Remove_Voxel</string>
   </property>
  </action>
  <action name="actionNew_Matriz">
   <property name="text">
    <string>New_Matriz</string>
   </property>
  </action>
  <action name="actionXY">
   <property name="text">
    <string>XY</string>
   </property>
  </action>
  <action name="actionZX">
   <property name="text">
    <string>ZX</string>
   </property>
   <property name="toolTip">
    <string>ZX</string>
   </property>
  </action>
  <action name="actionYZ">
   <property name="text">
    <string>YZ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawWidget</class>
   <extends>QWidget</extends>
   <header location="global">drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sliderRadius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>976</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>1046</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setR(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setG(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setB(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>789</x>
     <y>572</y>
    </hint>
    <hint type="destinationlabel">
     <x>1060</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderAxisX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelAxisX</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>522</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderAxisY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelAxisY</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderAxisZ</sender>
   <signal>valueChanged(int)</signal>
   <receiver>labelAxisZ</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonColor</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>pickColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>897</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>1065</x>
     <y>636</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInsert_Box</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionInsert_Box_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInsert_Elipse</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionInsert_Elipse_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInsert_Sphere</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionInsert_Sphere_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInsert_Voxel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionInsert_Voxel_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_Box</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionRemove_Box_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_Elipse</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionRemove_Elipse_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_Sphere</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionRemove_Sphere_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove_Voxel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_actionRemove_Voxel_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionXY</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedXY()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionYZ</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedYZ()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZX</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>clickedZX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setR(int)</slot>
  <slot>setG(int)</slot>
  <slot>setB(int)</slot>
  <slot>pickColor()</slot>
  <slot>on_actionInsert_Sphere_triggered()</slot>
  <slot>on_actionRemove_Sphere_triggered()</slot>
  <slot>on_actionInsert_Voxel_triggered()</slot>
  <slot>on_actionRemove_Voxel_triggered()</slot>
  <slot>on_actionInsert_Box_triggered()</slot>
  <slot>on_actionRemove_Box_triggered()</slot>
  <slot>on_actionInsert_Elipse_triggered()</slot>
  <slot>on_actionRemove_Elipse_triggered()</slot>
  <slot>clickedXY()</slot>
  <slot>clickedZX()</slot>
  <slot>clickedYZ()</slot>
 </slots>
</ui>
